catroid/src/androidTest/java/org/catrobat/catroid/test/formulaeditor/ParserTestOperators.java

```diff
@@ -336,4 +352,4 @@ public void testSubstraction() {
 		second.add(new InternToken(InternTokenType.NUMBER, "1"));
 		FormulaEditorTestUtil.testBinaryOperator(first, Operators.MINUS, second, Double.NaN, testSprite);
 	}
-}
+}
```

Comment:
[5fb02be02](https://github.com/Catrobat/Catroid/pull/2296/files/5fb02be0224a17a2c8ec0ce448fc80daaf565e5e) is a different kind of violation. [checkstyle.xml#L263-L264](https://github.com/Catrobat/Catroid/blob/f471d0c3abba61eaf03f56938909b7ea6219d5ca/catroid/config/checkstyle.xml#L263-L264). An empty line before closing braces is always unnecessary.

```
    }\n
\n <- violation
}\n
```

The violation commented on here is different. [checkstyle.xml#L236-L237](https://github.com/Catrobat/Catroid/blob/f471d0c3abba61eaf03f56938909b7ea6219d5ca/catroid/config/checkstyle.xml#L236-L237). Java files should end with a newline character. GitHub uses ![image](https://cloud.githubusercontent.com/assets/1794599/26059067/aa53f1ae-3980-11e7-8997-a2f9e4d7e0a1.png) to indicate that there is no newline character at the end.

```
    }\n
} <- violation
```